nodejs/week1/homework/src/backend/routes/cheap-meals.js

```diff
@@ -0,0 +1,21 @@
+const express = require("express");
+const router = express.Router();
+
+const meals = require("../data/meals.json");
+const reviews = require("../data/reviews.json");
+
+router.get("/", async(request, response) => {
```

Comment:
No need for `async` here (or in any other file in week1 homework) because you've already imported `meals`, so no need to `await` anything... :)